packages/common/container-definitions/src/deltas.ts

```diff
@@ -151,24 +168,38 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
 	 */
 	readonly hasCheckpointSequenceNumber: boolean;
 
-	/** Details of client */
+	/**
+	 * Details of client
+	 */
 	readonly clientDetails: IClientDetails;
 
-	/** Protocol version being used to communicate with the service */
+	/**
+	 * Protocol version being used to communicate with the service
+	 */
 	readonly version: string;
 
-	/** Max message size allowed to the delta manager */
+	/**
+	 * Max message size allowed to the delta manager
+	 */
 	readonly maxMessageSize: number;
 
-	/** Service configuration provided by the service. */
+	/**
+	 * Service configuration provided by the service.
+	 */
 	readonly serviceConfiguration: IClientConfiguration | undefined;
 
-	/** Flag to indicate whether the client can write or not. */
+	/**
+	 * Flag to indicate whether the client can write or not.
+	 */
 	readonly active: boolean;
 
 	readonly readOnlyInfo: ReadOnlyInfo;
 
-	/** Submit a signal to the service to be broadcast to other connected clients, but not persisted */
+	/**
+	 * Submit a signal to the service to be broadcast to other connected clients, but not persisted
+	 */
+	// TODO: use `unknown` instead (API breaking)
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Breaking because implementations of the interface could get compilation errors?

Reply:
Right